.github/workflows/build.yml

```diff
@@ -150,6 +150,10 @@ jobs:
           path: test-results/
           retention-days: 7
 
+      - name: Cleanup Predeploy for Docker Image
+        if: env.DOCKERIZE == 'true'
+        run: rm build/index.html build/sw.js build/manifest.json
```

Comment:
these were generated for testing purposes, remove them before they get included in the base image

Reply:
We have to add a line with `RUN rm -f build/index.html build/sw.js build/manifest.json` into the [pipeline.dockerfile](https://github.com/ParabolInc/parabol/blob/master/docker/parabol-ubi/docker-build/dockerfiles/pipeline.dockerfile) instead (and the rest of dockerfiles). That way, if someone builds it manually, it will build it correctly.

Should we then remove this step on the Build job?

Reply:
This can be removed from the build process and do not need to be included anywhere. The script web.js replaces the existing file with the filled one. I think you should remove it for simplicity.